homeassistant/block_async_io.py

```diff
@@ -1,5 +1,6 @@
 """Block I/O being done in asyncio."""
```

Comment:
Maybe rewrite the docstring, and log messages, a bit to encompass the added feature? It's not only I/O anymore but a blocking call.